corehq/tests/test_toggles.py

```diff
@@ -14,3 +14,12 @@ def test_toggle_properties():
         assert toggle.tag, 'Toggle "{}" tag missing'.format(toggle.slug)
         assert toggle.tag in ALL_TAGS, 'Toggle "{}" tag "{}" unrecognized'.format(toggle.slug, toggle.tag)
         assert toggle.namespaces, 'Toggle "{}" namespaces missing'.format(toggle.slug)
+
+
+def test_solutions_sub_tags():
+    """
+    Check Solutions sub-tags begin with 'Solutions - '
```

Comment:
Nice! I think I would add context here for the person who breaks the test, something like:

```
Client side toggle filtering logic currently depends on "Solutions" being in these tag names.
See https://github.com/dimagi/commcare-hq/pull/24575#discussion_r293995391
for context.
```